test/cli.test.ts

```diff
@@ -0,0 +1,39 @@
+describe('command line interface reads options correctly', () => {
```

Comment:
ESlint throws error: `ESLint: 'process' is not defined.(no-undef)`

To fix it, add next content to `.eslint.json` file:
```json
   "overrides": [
        {
            "files": ["test/**/*"],
            "env": {
                "jest": true,
                "node": true
            }
        }
    ],
```